src/common/tools/crawl.ts

```diff
@@ -45,26 +75,25 @@ export async function crawl(suite: string): Promise<TestSuiteListingEntry[]> {
 
       mod.g.validate();
```

Comment:
followup?: I think it may be possible to skip importing the module entirely when using `crawl` for runtime generation. We can skip validate(), and with #391 we don't need the description either. I suspect this would make it a lot faster.

Reply:
Oh it would be very, very nice. Right now we need to compile almost all the sources because crawl does the import. But if the listing just cares about file names, then we don't need to do real imports. And we can get away with a Node.js build just for stuff in tools/ (maybe some other utilities too).

Reply:
did this now.